features/cellular/framework/AT/AT_CellularContext.cpp

```diff
@@ -112,6 +113,11 @@ void AT_CellularContext::enable_hup(bool enable)
     }
 }
 
+CellularDevice *AT_CellularContext::get_device() const
```

Comment:
This is returning base class member, so you can use base class get_device() as such. This is only extra code and therefore not needed.

Reply:
There is a type-safety reason to have it - the method should exist, but be returning `AT_CellularDevice *`.

Reply:
@AnttiKauppila is the current implementation ok? It's done as Kevin suggested.